kedro/framework/cli/starters.py

```diff
@@ -488,7 +494,7 @@ def _select_prompts_to_display(
         del prompts_required["add_ons"]
 
     if project_name is not None:
-        if bool(re.match(r"^[\w -]{2,}$", project_name)) is False:
+        if not re.match(r"^[\w -]{2,}$", project_name):
```

Comment:
the boolean expression is not needed. `re.match` either return an object or `None`